tick/optim/model/src/coxreg_partial_lik.h

```diff
@@ -9,8 +9,7 @@
 
 #include "model.h"
 
-
-class ModelCoxRegPartialLik : public Model {
+class DLL_PUBLIC ModelCoxRegPartialLik : public Model {
```

Comment:
If I understand correctly, this flag must be put if the class is shared with another module. It this class shared out of model module?